build.gradle

```diff
@@ -509,6 +509,23 @@ task runProductionAutoTestServer(type: JavaExec, dependsOn: [remapJar, remapTest
 	}
 }
 
+def addPomMetadataInformation(MavenPom pom) {
```

Comment:
Would be nice to also set name and url. description could possibly be driven by the fabric.mod.json file, but thats a little more effort.